care/hcx/api/viewsets/listener.py

```diff
@@ -89,3 +92,33 @@ def post(self, request, *args, **kwargs):
         )
 
         return Response({}, status=status.HTTP_202_ACCEPTED)
+
+
+class CommunicationRequestView(GenericAPIView):
```

Comment:
How are these authorised ?

Reply:
This is actually handled using a custom authentication class (https://github.com/coronasafe/care/blob/b64538337000442573be7fda175893b37dbb7447/config/authentication.py#L159) in a different pr (https://github.com/coronasafe/care/pull/1229/files)

Reply:
Can we have all related changes in a single PR?